bin/send-to-mailing-list

```diff
@@ -68,27 +68,13 @@ def process_command_line(argv=ARGV)
 end
 
 @contacts_lock = Mutex.new
-@contacts_cache = nil
```

Comment:
It's faster not to load the entire DB of contacts at startup, yes. Especially when sending to a small list. 

Then, since all the csv files are deduped already, we will never have a cache hit. Therefore it doesn't even make sense to cache them lazily as we go. But I'd be happy to add it back in that form. My main concern was removing the horrendously long delay at startup.